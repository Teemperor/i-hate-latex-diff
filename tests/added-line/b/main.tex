
\documentclass[12pt]{article}
\begin{document}
Hello world!
This got added
$Hello world!$ %math mode 
\end{document}
