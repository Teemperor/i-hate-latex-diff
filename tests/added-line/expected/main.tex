
\documentclass[12pt]{article}
\begin{document}
Hello world!
\addedChange{This }\addedChange{got }\addedChange{added}
$Hello world!$ %math mode 
\end{document}
